.github/workflows/@sync-docs.yml

```
@@ -0,0 +1,76 @@
+name: Sync Docs to Vector Store (PR & Push)
+
+on:
+  workflow_dispatch
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)